src/sdk/pynni/nni/nas/pytorch/mutables.py

```diff
@@ -109,11 +112,12 @@ def forward(self, optional_inputs, semantic_labels=None):
             "Length of the input list must be equal to number of candidates."
         if semantic_labels is None:
             semantic_labels = ["default_label"] * self.n_candidates
-        out, mask = self.mutator.on_forward(self, optional_inputs, semantic_labels)
+        out, mask = self.mutator.on_forward(
+            self, optional_inputs, semantic_labels)
```

Comment:
why?

Reply:
Auto format. We can use same tool (vs code?), or have the same line width setting.

Reply:
You might need to set line width to 140, which is defined in our pylint.